server/sonar-web/src/main/js/components/source-viewer/popups/line-actions-popup.js

```diff
@@ -29,7 +29,7 @@ export default Popup.extend({
 
   getPermalink (e) {
     e.preventDefault();
-    const url = `/component/index?id=${encodeURIComponent(this.model.key())}&line=${this.options.line}`;
+    const url = `${window.baseUrl}/component/index?id=${encodeURIComponent(this.model.key())}&line=${this.options.line}`;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 121 characters long line (which is greater than 120 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3ALineLength)